src/plugins/mutualGroupDMs.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Vencord, a modification for Discord's desktop app
+ * Copyright (c) 2023 Vendicated and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+import { findByPropsLazy } from "@webpack";
+import { Avatar,ChannelStore, Clickable, RelationshipStore, ScrollerThin, UserStore } from "@webpack/common";
+import { Channel, User } from "discord-types/general";
+
+const SelectedChannelActionCreators = findByPropsLazy("selectPrivateChannel");
+const AvatarUtils = findByPropsLazy("getChannelIconURL");
+const UserUtils = findByPropsLazy("getGlobalName");
+
+const ProfileListClasses = findByPropsLazy("emptyIconFriends", "emptyIconGuilds");
+const GuildLabelClasses = findByPropsLazy("guildNick", "guildAvatarWithoutIcon");
+
+function getGroupDMName(channel: Channel) {
+    return channel.name || channel.recipients.map(UserStore.getUser).filter(x => x != null).map(x =>
+        RelationshipStore.getNickname(x.id) || UserUtils.getName(x)
+    ).join(", ");
+}
+
+export default definePlugin({
+    name: "MutualGroupDMs",
+    description: "Shows mutual group dms on profile.",
+    authors: [Devs.amia],
+
+    patches: [
+        {
+            find: ".Messages.USER_PROFILE_MODAL", // Note: the module is lazy-loaded
+            replacement: [
+                {
+                    match: /(?<=(\i)\.isClyde\(\)\?null:(\(0,\i\.\i\).+?id:)\i\.\i\.MUTUAL_GUILDS(,children:)\i\.\i\.Messages\.MUTUAL_GUILDS}\))/,
+                    replace: ",!$1.bot?$2\"MUTUAL_GDMS\"$3\"Mutual Groups\"}):null"
+                },
+                {
+                    match: /({user:(\i),onClose:(\i)}\);)(case \i\.\i\.MUTUAL_FRIENDS)/,
```

Comment:
no need to match either of these
```suggestion
                    match: /(?<={user:(\i),onClose:(\i)}\);)(?=case \i\.\i\.MUTUAL_FRIENDS)/,
```